src/show.jl

```diff
@@ -132,26 +133,51 @@ const tensorboard = Ref{Tensorboard}()
 
 function Base.close(board::Tensorboard)
     kill(board.proc, 2)
-    #rm(board.logdir, recursive=true, force=true)
 end
 
-function get_tensorboard(logdir=nothing)
-    if true#!isdefined(tensorboard, :x) || (logdir !== nothing && tensorboard[].logdir ≠ logdir)
-        if isdefined(tensorboard, :x)
-            port = tensorboard[].port + 1
-        else
-            port = 6006
-        end
-        if logdir === nothing
-            logdir = mktempdir()
-        end
-        _, proc = open(`tensorboard --logdir=$logdir --port=$port`)
-        tensorboard[] = Tensorboard(proc, logdir, port)
-        atexit() do
-            close(tensorboard[])
+"""
+    find_tensorboard()
+
+Return the path to the tensorboard executable.
+
+Checks everywhere in the system path, as well in the binary python directory
+associated with the Conda.jl installation, should it exist.
+"""
+function find_tensorboard()
+    path = nothing
```

Comment:
Do `local path` rather than `path=nothing`
so it can be type stable?
Or use a nullable?
I guess with the small union optimisation this is less of a thing now, though


`local path` would require moving the errorcheck into this function which I think might be cleaner anyway.